datahub-web-react/src/app/entity/dataset/shared/TagSummary.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useGetTagQuery } from '../../../../graphql/tag.generated';
+import { HoverEntityTooltip } from '../../../recommendations/renderer/component/HoverEntityTooltip';
+import { StyledTag } from '../../shared/components/styled/StyledTag';
+
+const TagLink = styled.span`
+    display: inline-block;
+`;
+
+type Props = {
+    urn: string;
+};
+
+export const TagSummary = ({ urn }: Props) => {
+    const { data } = useGetTagQuery({ variables: { urn } });
+    return (
+        <HoverEntityTooltip>
+            <TagLink key={data?.tag?.urn}>
```

Comment:
so this should actually be a link to the tag page - so if you look at `TagTermGroup.tsx` you can export the `TagLink` component and simply import that here! you'll need to set the link like we do in that component.

Reply:
Fixed and committed!!